pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -343,6 +343,20 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 		return nil
 	}
 
+	// it is possible that reclaimNodeLevelResources suceeded but didn't realise. Let's check the actual situation
+	// before we start evicting pods
+	observations, _, err = makeSignalObservations(m.summaryProvider, capacityProvider, activePods)
+	if err != nil {
+		glog.Errorf("eviction manager: unexpected err: %v", err)
+		return nil
+	}
+
+	debugLogObservations("observations", observations)
+	if len(thresholdsMet(m.thresholdsMet, observations, true)) == 0 {
```

Comment:
I think you need to use `thresholds` instead of `m.thresholdsMet` so that it only considers those which previously had updated stats.